sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -195,6 +195,17 @@ def test_constant_batch(self):
           | beam.Map(len))
       assert_that(res, equal_to([10, 10, 10, 5]))
 
+  def test_constant_batch_no_metrics(self):
+    # Assumes a single bundle...
+    with TestPipeline() as p:
+      res = (
+          p
+          | beam.Create(range(35))
+          | util.BatchElements(min_batch_size=10, max_batch_size=10,
+                               record_metrics=False)
+          | beam.Map(len))
+      assert_that(res, equal_to([10, 10, 10, 5]))
```

Comment:
Could you make this test verify that the transform did not create metrics? Let me know if you need help with that.

Reply:
I updated both tests, PTAL!